modules/services/crashlogging/src/main/java/au/com/shiftyjelly/pocketcasts/crashlogging/PocketCastsCrashLoggingDataProvider.kt

```diff
@@ -70,7 +65,15 @@ class PocketCastsCrashLoggingDataProvider @Inject constructor(
         return false
     }
 
+    override val errorSampling: ErrorSampling =
+        if (BuildConfig.BUILD_PLATFORM == "mobile") {
+            ErrorSampling.Enabled(MOBILE_ERROR_SAMPLING)
+        } else {
+            ErrorSampling.Disabled
+        }
+
     companion object {
         const val GLOBAL_TAG_APP_PLATFORM = "app.platform"
+        const val MOBILE_ERROR_SAMPLING = 0.3
```

Comment:
Discussed in previous PR: https://github.com/Automattic/pocket-casts-android/pull/2152#discussion_r1587509155

Related Tracks PR: https://github.com/Automattic/Automattic-Tracks-Android/pull/217